packages/react-spinbutton/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -71,29 +73,17 @@ export type SpinButtonCommons = {
   /**
    * Difference between two adjacent values of the control.
    * This value is used to calculate the precision of the input if no `precision` is given.
-   * The precision calculated this way will always be \>= 0.
+   * The precision calculated this way will always be greater than or equal 0.
    * @defaultvalue 1
```

Comment:
nit: I think this should be @defaultValue

Reply:
Yeah, it should be `@default`: https://github.com/microsoft/fluentui/pull/18566#discussion_r681797428